ouroboros-consensus-shelley/test/Test/Consensus/Shelley/Ledger/Golden.hs

```diff
@@ -42,6 +52,7 @@ import           Test.Tasty.HUnit
 import           Test.Util.Golden
 import           Test.Util.Orphans.Arbitrary ()
 
+import           Test.Cardano.Crypto.VRF.Fake
```

Comment:
Can you add an explicit import list?
```suggestion
import           Test.Cardano.Crypto.VRF.Fake

```
Separate it from the line below so that it's clear that it's not from the same package as the line below, which is from the package we're in.